site/layouts/shortcodes/orange-global-headers.html

```diff
@@ -92,7 +92,8 @@ <h1 class="title">Title</h1>
 {{- if eq $mode "search" }}
     <form class="d-flex d-none d-lg-flex col-3 navbar-item" role="search">
       <!-- todo : reforge when .dark-input -->
-      <input class="form-control bg-transparent text-white border border-2 border-dark" type="search" placeholder="Search" aria-label="Search">
+      <label for="search-input" class="visually-hidden">Search</label>
```

Comment:
Just changes something that was a bit weird for accessibility

Reply:
Already checked with a11y team?

Reply:
It doesn't seem to be modified in `orange-navbar.md` > With search box

Reply:
Should we change it for `navbar.md` too ?

Reply:
If so, I think there might be some changes on Bs to try

Reply:
Asked in https://github.com/twbs/bootstrap/issues/21842.